department-of-veterans-affairs/caseflow
# jHoang/APPEALS-17523

Resolves [17523](https://vajira.max.gov/browse/APPEALS-17523?workflowName=VA+Caseflow+Sandbox+Story+Enabler+Workflow&stepId=20)

### Description
Added a banner to the special issues view alerting users that the MST Checkbox and PACT Checkbox will be handled at issue level now.

### Acceptance Criteria

### Testing Plan
1. Login as an attorney or judge that has a legacy case assigned. (BVASCASPER1 or BVAAABSHIRE)
2. Go to a legacy appeal with at least one issue.
3. Under the Actions dropdown click on "Decision ready for review" (for attorneys)
OR
Under the Actions dropdown click on "Ready for dispatch" (for judges)
4. Observe that a blue banner appears at the top of the page with a description that matches the [wireframe](https://www.figma.com/proto/va7aL4f0BPdh5LcPO0I8O6/PI-12-(2023)%3A-Intake?node-id=135-25009&scaling=min-zoom&page-id=0%3A42964&starting-point-node-id=135%3A21345)

### User Facing Changes

 BEFORE
<img width="1719" alt="Screenshot 2023-04-18 at 1 26 37 PM" src="https://user-images.githubusercontent.com/111081469/232871165-08c833cf-4cb6-43fd-9090-9f8e426dd690.png">

AFTER

<img width="1719" alt="Screenshot 2023-04-18 at 1 34 24 PM" src="https://user-images.githubusercontent.com/111081469/232871351-f5f97238-65e7-4c52-a3c1-161e6c05b33b.png">
